src/userAccount/views/OrderDetails/Page.tsx

```diff
@@ -26,12 +26,12 @@ const extractOrderLines = (lines: OrderDetail_lines[]): ILine[] => {
         ...line.unitPrice,
         currency: line.unitPrice.currency,
         gross: {
-          amount: line.quantity * line.unitPrice.gross.amount,
           ...line.unitPrice.gross,
+          amount: line.quantity * line.unitPrice.gross.amount,
```

Comment:
It should be taken from API

Reply:
Oh, I didn't know it is available in API. Fixed - update in https://github.com/mirumee/saleor-sdk/pull/32

Reply:
I know that it needs sdk dependency update, but sdk with update is not yet released